Shared/Views/CampaignView.swift

```diff
@@ -53,8 +53,8 @@ struct CampaignView: View {
         }
     }
     
-    var description: String {
-        fundraisingEvent?.description ?? initialCampaign?.description ?? ""
+    var description: LocalizedStringKey {
+        LocalizedStringKey(fundraisingEvent?.description ?? initialCampaign?.description ?? "")
```

Comment:
`AttributedString(markdown:` might be a "more correct" way to do this?

Reply:
Learn something new every day!